src/applications/gi/sass/partials/_gi-vet-tec.scss

```diff
@@ -28,4 +28,8 @@
       display: none !important; }
   }
 
+  .vettec-sco-list {
+    list-style: none;
```

Comment:
I'm not sure that styling the `<ul>`s to not appear as lists is ok.  The design system has some [guidance on bulleted lists](https://design.va.gov/content-style-guide/bulleted-lists).

@1Copenut is this ok from your perspective?

Reply:
This was the recommended fix by @1Copenut to resolve a 508 issue. Since the Primary and Secondary contact list is in fact a list, he felt it necessary that a screenreader could announce it as such.

Reply:
@bkjohnson I'm okay with these being lists without bullets. Since these are conveying a lot of information, they're more like cards, so it's borderline a new pattern. I'm going to add these as an example to a design team question about card patterns for awareness.

Great question, thank you for the heads up.